src/components/minimap/minimap.js

```diff
@@ -402,6 +404,7 @@ const emptyGraphSize = {};
 
 export const mapStateToProps = (state, ownProps) => ({
   visible: state.visible.miniMap,
```

Comment:
This is confusing to be honest. Since visible has other props, we should rename this to miniMapVisible or miniMapVisibleState

Reply:
Yes it is confusing.